src/ReactiveDomain.Identity/Stores/UserStore.cs

```diff
@@ -20,7 +21,7 @@ public class UserStore
 
         public UserStore(IConfiguredConnection conn)
         {
-            _userSvc = new UserSvc(conn.GetRepository());
+            _userSvc = new UserSvc(conn.GetRepository(), new Dispatcher("bus-to-nowhere"));
```

Comment:
Can we make this a `NullBus` instead, so as to be more explicit about it being unused here?

Reply:
null bus throws on attempted subscription :-(

Reply:
oh